libs/libc/stdio/lib_getdelim.c

```diff
@@ -33,33 +34,40 @@
  ****************************************************************************/
 
 /* Some environments may return CR as end-of-line, others LF, and others
- * both.  Because of the definition of the getline() function, it can handle
- * only single character line terminators.
+ * both.
  */
 
-#undef HAVE_GETLINE
 #if defined(CONFIG_EOL_IS_CR)
```

Comment:
can we remove CONFIG_EOL_IS_xxx and convert all possible combination dynamically?